### DISCUSSION TITLE
Is there any way I can recover an account I've lost?

### DISCUSSION THREAD

**Discussion by Bonavigoo** [community/community#119720] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Well, I'm a genius: I've lost my .txt file with the code, and my Microsoft Authenticator app generates invalid codes (I've bought another cellphone, since my old one broke).
Basically, I have a (broken) Authenticator and my account logged into GitHub Desktop.
Is there any way I can recover the account, or it's lost for good?

**Comment by jvmhonorato** [comment]
read this:
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/unlinking-your-email-address-from-a-locked-account

**Comment by ebndev** [comment]
Hi @Bonavigoo,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
